docs/latest_release_notes.md

```diff
@@ -0,0 +1,604 @@
+# Latest release notes
```

Comment:
Isn't this file automatically generated by the update_changelog script?

The script should probably be invoked during build, and the generated page not be part of the repo... right?

Reply:
update_changelog script does a `head -1` then adds to this markdown.